src/test/unit/algorithms/BacktrackingTMCppTest.cpp

```diff
@@ -29,15 +29,19 @@
  *
  * For those not familiar with GTest:
  * ASSERT_TRUE(value)   -- Fatal assertion that the value is true. Test
- *terminates if false. ASSERT_FALSE(value)   -- Fatal assertion that the value
```

Comment:
maybe remove these? GTest is used everywhere here, people are either familiar, or can find the meanings?